backend/Makefile

```diff
@@ -66,6 +66,7 @@ $(TEST_ACCEPTANCE_TARGETS):
 	@$(MAKE) -C services/$(subst -test-acceptance,,$@) test-acceptance-run
 
 .PHONY: test-integration
+test-integration: MENDER_IMAGE_TAG=$(MENDER_IMAGE_TAG_TEST)
```

Comment:
do we need export here?

Reply:
No, the target is not forking another make process.

Reply:
Sorry, you're totally right! I didn't think of the variable not being exposed to the script this way.